buildSrc/src/main/kotlin/boudicca-base.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id("boudicca-base")
+}
+
+group = "base.boudicca"
+version = "0.1.0-SNAPSHOT"
```

Comment:
If the version is set here, then it seems it's not correctly passed for the generation of openapi.json files

Reply:
maybe, but that was wrong with your changes as well. i really want to close this MR soon because it is way too big and blocks pretty much everything else